.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@antleblanc @cbourgois @frenautvh @jleveugle
+*  @antleblanc @cbourgois @frenautvh @jleveugle
```

Comment:
Is there a way for everyone working on the manager to get notifications even though they are not code owners ? It would help us follow track of how the project is going forward

Reply:
These links might help you:

https://github.com/pulls
https://github.com/notifications

As we are improving those critical pieces, we tend reduce the number of people involved.